examples/jsm/animation/AnimationClipCreator.js

```diff
@@ -92,7 +92,7 @@ class AnimationClipCreator {
 	static CreateMaterialColorAnimation( duration, colors ) {
 
 		const times = [], values = [],
-			timeStep = duration / colors.length;
+			timeStep = duration / ( colors.length - 1 );
```

Comment:
Do you mind fixing the edge case when `colors.length` is `1`? E.g. 
```js
timeStep = ( colors.length > 1 ) ? duration / ( colors.length - 1 ) : 0;
```
Right now, `timeStep` ends up `Infinity` which produces `NaN` values later in the code.

If there is only one value, `times` should be `0`.